src/elements/hero/src/index.tsx

```diff
@@ -42,7 +42,7 @@ const Hero: React.FC<Props> = ({
   className
 }) => {
   const { theme }: any = useThemeUI()
-  const breadcrumbVariant = get(theme, 'hero.breadcrumbs.variant')
+  const breadcrumbVariant = get(theme, 'elements.hero.breadcrumbs.variant')
```

Comment:
barney is updating this depdendency anyway so get him to update to the version that he is updating to

Reply:
This is a bug, was missed during migration so this is 👍